CoreFeatures/Database-postgresql/PostgresAddIndexesTask/src/test/java/info/smart_tools/smartactors/database_postgresql/postgres_add_indexes_task/PostgresAddIndexesTaskTest.java

```diff
@@ -0,0 +1,137 @@
+import info.smart_tools.smartactors.database_postgresql.postgres_connection.JDBCCompiledQuery;
+import info.smart_tools.smartactors.database_postgresql.postgres_connection.QueryStatement;
+import info.smart_tools.smartactors.feature_loading_system.bootstrap.Bootstrap;
+import info.smart_tools.smartactors.feature_loading_system.interfaces.ibootstrap.exception.ProcessExecutionException;
+import info.smart_tools.smartactors.feature_loading_system.interfaces.iplugin.exception.PluginException;
+import info.smart_tools.smartactors.iobject.ds_object.DSObject;
+import info.smart_tools.smartactors.iobject.iobject.exception.ReadValueException;
+import info.smart_tools.smartactors.iobject_plugins.dsobject_plugin.PluginDSObject;
+import info.smart_tools.smartactors.iobject_plugins.ifieldname_plugin.IFieldNamePlugin;
+import info.smart_tools.smartactors.ioc.iioccontainer.exception.RegistrationException;
+import info.smart_tools.smartactors.ioc.iioccontainer.exception.ResolutionException;
+import info.smart_tools.smartactors.ioc.ioc.IOC;
+import info.smart_tools.smartactors.ioc.key_tools.Keys;
+import info.smart_tools.smartactors.ioc_plugins.ioc_keys_plugin.PluginIOCKeys;
+import info.smart_tools.smartactors.ioc_plugins.ioc_simple_container_plugin.PluginIOCSimpleContainer;
+import info.smart_tools.smartactors.task.interfaces.itask.exception.TaskExecutionException;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+import static org.junit.Assert.fail;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests for PostgresAddIndexesTask.
+ */
+public class PostgresAddIndexesTaskTest {
+
+    private IDatabaseTask task;
+    private IStorageConnection connection;
+    private AddIndexesMessage message;
+    private JDBCCompiledQuery compiledQuery;
+    private PreparedStatement sqlStatement;
+
+    @BeforeClass
+    public static void prepareIOC() throws PluginException, ProcessExecutionException {
+        Bootstrap bootstrap = new Bootstrap();
+        new PluginIOCSimpleContainer(bootstrap).load();
+        new PluginIOCKeys(bootstrap).load();
+        new IFieldNamePlugin(bootstrap).load();
+        new PluginDSObject(bootstrap).load();
+        bootstrap.start();
+    }
+
+    @Before
+    public void setUp() throws QueryBuildException, InvalidArgumentException, ResolutionException, RegistrationException, ReadValueException, StorageException, SQLException {
+        sqlStatement = mock(PreparedStatement.class);
+
+        compiledQuery = mock(JDBCCompiledQuery.class);
+        when(compiledQuery.getPreparedStatement()).thenReturn(sqlStatement);
+
+        connection = mock(IStorageConnection.class);
+        when(connection.compileQuery(any())).thenReturn(compiledQuery);
+
+        task = new PostgresAddIndexesTask(connection);
```

Comment:
Probably, we also need the same test for `PostgresAddIndexesSafeTask`.

Reply:
I changed the class in test to PostgresAddIndexesSafeTask. Since PostgresAddIndexesSafeTask calls PostgresAddIndexesTask with minimal wrapper then it tests both.